src/assets/locales/fr.json

```diff
@@ -321,22 +354,23 @@
   "people-sidebar.person-name.you": "Vous",
   "people-sidebar.presence.entering": "Entrée dans la salle",
   "people-sidebar.presence.in-lobby": "Dans le hall d'attente",
-  "people-sidebar.presence.in-room": "Dans la Salle",
+  "people-sidebar.presence.in-room": "Dans la salle",
   "people-sidebar.title": "Personnes ({numPeople})",
   "people-sidebar.voice-label.muted": "Micro coupé",
-  "people-sidebar.voice-label.not-talking": "Ne pas parler",
-  "people-sidebar.voice-label.talking": "Parler",
-  "photo-message.body": "{name} prendre une photo <a>photo</a>.",
-  "photo-message.default-tweet": "Capturé dans {shareHashtag}, rejoignez-moi ici {url}",
+  "people-sidebar.voice-label.not-talking": "Ne parle pas",
+  "people-sidebar.voice-label.talking": "Parle",
+  "photo-message.body": "{name} a pris une <a>photo</a>.",
+  "photo-message.default-tweet": "Capturé dans {shareHashtag}",
   "place-popover.item-type.avatar": "Avatar",
-  "place-popover.item-type.camera": "Camera",
+  "place-popover.item-type.camera": "Caméra",
   "place-popover.item-type.gif": "GIF",
   "place-popover.item-type.model": "Modèle 3D",
   "place-popover.item-type.pen": "Stylo",
   "place-popover.item-type.scene": "Scène",
   "place-popover.item-type.upload": "Téléverser",
   "place-popover.title": "Placer",
   "preferences-screen.browser-default": "Réglages par défaut du navigateur",
+  "preferences-screen.category.accessibility": "Accessibilité",
   "preferences-screen.category.audio": "Audio",
   "preferences-screen.category.controls": "Contrôles",
```

Comment:
"Commandes" ?

Reply:
I don't know how controls is translated into games in French, is it contrôles or commandes? I'll like a second opinion. :)
If we change it, we need to change it in
"more-menu.controls": "Contrôles"

Reply:
Just read your first comment, ok I'll change to "Commandes"